pkg/logs/config/config.go

```diff
@@ -74,8 +81,12 @@ func BuildEndpoints() (*client.Endpoints, error) {
 		useSSL = true
 	default:
 		// datadog settings
-		main.Host = coreConfig.Datadog.GetString("logs_config.dd_url")
-		main.Port = coreConfig.Datadog.GetInt("logs_config.dd_port")
+		main.Host = coreConfig.GetMainEndpoint(logsAgentPrefix, "logs_config.dd_url")
```

Comment:
Aren't we making the `site` option work for eu only?

Reply:
No because if `site` is configured to us, `found` below will be evaluated to false and we will be in this case https://github.com/DataDog/datadog-agent/pull/2824/files#diff-945316aac270bbb351b00cccdfc89e68R88

which default to https://github.com/DataDog/datadog-agent/pull/2824/files#diff-091ace02b49a46f55f2586cfec89a160R307

The behaviour is also tested here https://github.com/DataDog/datadog-agent/pull/2824/files#diff-f35e02ce8483c193319357d3d29e2b07R53

Reply:
OK but 
1) could we make `logsEuEndpoints` a bit more generic, for when we'll have more endpoints?
2) even if it works, isn't it a but clearer to have us explicitly in that list?